linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml

```
@@ -61,19 +59,6 @@ warnings:
     - general: |-
         The system needs to be rebooted for these changes to take effect.
     - regulatory: |-
-        System Crypto Modules must be provided by a vendor that undergoes
-        FIPS-140 certifications.
-        FIPS-140 is applicable to all Federal agencies that use
-        cryptographic-based security systems to protect sensitive information
-        in computer and telecommunication systems (including voice systems) as
-        defined in Section 5131 of the Information Technology Management Reform
-        Act of 1996, Public Law 104-106. This standard shall be used in
-        designing and implementing cryptographic modules that Federal
-        departments and agencies operate or are operated for them under
-        contract. See <b>{{{ weblink(link="https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.140-2.pdf") }}}</b>
-        To meet this, the system has to have cryptographic software provided by
-        a vendor that has undergone this certification. This means providing
-        documentation, test results, design information, and independent third
-        party review by an accredited lab. While open source software is
-        capable of meeting this, it does not meet FIPS-140 unless the vendor
-        submits to this process.
+        This rule DOES NOT CHECK if the components of the operating system are FIPS certified.
+        You can find the list of FIPS certified modules at {{{ weblink(link="https://csrc.nist.rip/groups/STM/cmvp/documents/140-1/1401vend.htm") }}}.
```

```suggestion
        You can find the list of FIPS certified modules at {{{ weblink(link="https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search") }}}.
```

This list seems rather outdated. I believe the suggested page allows you to search the NIST database for most up to date information about certification of cryptographic modules